lib/widgets/content.dart

```diff
@@ -750,9 +771,14 @@ final _kCodeBlockStyle = kMonospaceTextStyle
 // const _kInlineCodeRightBracket = '⟩';
 
 class UserMention extends StatelessWidget {
-  const UserMention({super.key, required this.node});
+  const UserMention({
+    super.key,
+    required this.node,
+    required this.surroundingTextStyle,
+  });
 
   final UserMentionNode node;
+  final TextStyle surroundingTextStyle;
```

Comment:
nit: I think of the `node` field of these content widgets as somewhat akin to `child` — it's the payload of the widget, describing what content it should show, whereas this other field is some metadata about how to show it. So let's put the node at the end.

(From a quick search for "node;" in this file, it looks like these are actually the first examples of a content widget that takes any other field in addition to a "node".)